### PR TITLE
OSDOCS#11143: Multiple vCenter support for vSphere CSI

### PR DESCRIPTION
**Pull request by lpettyjo**
Version(s): 4.17+

Issue: https://issues.redhat.com/browse/OSDOCS-11143

Link to docs preview: https://81394--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-vsphere.html#persistent-storage-csi-vsphere-multi-vcenter-support-overview_persistent-storage-csi-vsphere

QE review:
- [x] QE has approved this change.

Additional information:

Assuming install only option with no day 2 ability for now.

**PTAL**: @jsafrane @duanwei33 @gcharot

### CODE REVIEW

**Review by tmalove — APPROVED** [review]

**Comment by duanwei33** [comment]
LGTM from QE side

**Comment by gcharot** [comment]
LGTM if all the configuration steps are described in the OCP vsphere installation doc

File: modules/persistent-storage-csi-vsphere-multi-vcenter-support-procedure-install.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+// storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-multi-vcenter-support-procedure-install_{context}"]
+= Configuring multiple vCenters during installation
+
+To configure multiple vCenters during installation:
+
+* Specify multiple vSphere clusters during installation. See xref:../../installing/installing_vsphere/installation-config-parameters-vsphere.adoc[Installation configuration parameters for vSphere].
```
Comment (jsafrane) [inline modules/persistent-storage-csi-vsphere-multi-vcenter-support-procedure-install.adoc:12]: The linked page says "*Currently, only a single vCenter server is supported*", without any TechPreview note about three vCenters.

Is someone else updating the install docs?

Comment (lpettyjo) [reply modules/persistent-storage-csi-vsphere-multi-vcenter-support-procedure-install.adoc:12]: Yes. I will ask them about getting this fixed and leave this comment open until it's confirmed.

File: modules/persistent-storage-csi-vsphere-multi-vcenter-support-procedure-install.adoc [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+//
+// storage/container_storage_interface/persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-multi-vcenter-support-procedure-install_{context}"]
+= Configuring multiple vCenters during installation
+
+To configure multiple vCenters during installation:
+
```
Comment (gnufied) [inline modules/persistent-storage-csi-vsphere-multi-vcenter-support-procedure-install.adoc:11]: Do we need to add wording that - multiple vcenters can't be added as day-2 operation?

File: modules/persistent-storage-csi-vsphere-multi-vcenter-support-procedure-install.adoc (hunk 1 above)
Comment (snarayan-redhat) [reply modules/persistent-storage-csi-vsphere-multi-vcenter-support-procedure-install.adoc:12]: This change was [merged](https://docs.openshift.com/container-platform/4.17/installing/installing_vsphere/installation-config-parameters-vsphere.html#installation-configuration-parameters-additional-vsphere_installation-config-parameters-vsphere) yesterday.

Comment (lpettyjo) [reply modules/persistent-storage-csi-vsphere-multi-vcenter-support-procedure-install.adoc:12]: Resolved.

File: modules/persistent-storage-csi-vsphere-multi-vcenter-support-procedure-install.adoc (hunk 2 above)
Comment (lpettyjo) [reply modules/persistent-storage-csi-vsphere-multi-vcenter-support-procedure-install.adoc:11]: @gnufied I've added more explicit information about this in the Over section that is just above the procedure. Let me know if that is sufficient.

**Comment by gnufied** [comment]
lgtm from storage side.

**Pull request merged**